tasks/skylab/Metrics.wdl

```diff
@@ -8,8 +8,7 @@ task CalculateCellMetrics {
     String input_id
 
     # runtime values
-    #String docker = "us.gcr.io/broad-gotc-prod/warp-tools:1.0.9-1700252065"
-    String docker = "us.gcr.io/broad-gotc-prod/warp-tools:2.0.0"
+    String docker = "us.gcr.io/broad-gotc-prod/warp-tools:aa-mt-tagsort"
```

Comment:
do you mind rebuilding the docker using the `warp-tools:1.0.9-blahblahblah` nomenclature and using that here instead of `aa-mt-tagsort`?

Reply:
Yes of course. Im going to rebuild the docker with an actual version number once the warp-tools PR gets merged.